search-commons/src/test/resources/datasource_urls_ticket.json

```
@@ -13,7 +15,7 @@
   "ID_NOT=018de5060dd0-3e2570f5-3d56-4479-b79e-4410b81626c7,018de5044416-7a3b7a74-978f-45fb-a743-c9c0b0d39e65": 18,
   "MESSAGES=en+DOI": 1,
   "MESSAGES_NOT=en+DOI": 19,
-  "MODIFIED_DATE=1872-01-01,2024": 1,
+  "MODIFIED_DATE=1872-01-01,2023": 1,
```

Add a test for the last case in expandDateFirst/Last() methods?

```suggestion
  "MODIFIED_DATE=1872-01-01,2023": 1,
  "MODIFIED_DATE=1872-01-01,2023-05": <insert correct number here>,
```